yazi-core/src/input/snap.rs

```diff
@@ -35,6 +35,9 @@ impl InputSnap {
 		self.cursor = self.cursor.min(self.value.chars().count().saturating_sub(self.mode.delta()));
 		self.offset =
 			self.offset.min(self.cursor.saturating_sub(Self::find_window(&self.rev(), 0, limit).end));
+		if let Some((_base, ext)) = self.value.rsplit_once('.') {
```

Comment:
I think it's pointless to move the cursor before the dot in filenames like `.bashrc`

Reply:
That makes sense - code updated :)

I also used `saturating_sub` instead of just `-`  because I got a crash otherwise (not sure why)